### PR TITLE
Upgrade remote-network resource - part 5

### PR DESCRIPTION
**Pull request by vmanilo**
Resolves upgrade provider

Test passes on fork: https://github.com/vmanilo/terraform-provider-twingate/actions/runs/5825601625

## Changes
- upgrade remote-network resource

### CODE REVIEW

File: docs/data-sources/remote_networks.md [hunk 1]
```diff
@@ -21,14 +21,14 @@ data "twingate_remote_networks" "all" {}
...
+- `remote_networks` (Attributes List) List of Remote Networks (see [below for nested schema](#nestedatt--remote_networks))
 
-<a id="nestedblock--remote_networks"></a>
+<a id="nestedatt--remote_networks"></a>
 ### Nested Schema for `remote_networks`
 
 Read-Only:
 
-- `id` (String) The ID of the Remote Network
+- `id` (String) The ID of the Remote Network resource.
```
Comment (vmanilo) [inline docs/data-sources/remote_networks.md:32]: @ekampf should I revert this changes?

Comment (ekampf) [reply docs/data-sources/remote_networks.md:32]: yes

File: docs/data-sources/remote_networks.md [hunk 2]
```diff
@@ -21,14 +21,14 @@ data "twingate_remote_networks" "all" {}
 
 ### Read-Only
 
-- `id` (String) The ID of this resource.
-- `remote_networks` (Block List) List of Remote Networks (see [below for nested schema](#nestedblock--remote_networks))
+- `id` (String) The ID of the data source result
```
[the remark this answers is not in the document]
Comment (ekampf) [reply docs/data-sources/remote_networks.md:24]: yes

**Review by ekampf — APPROVED** [review]

File: docs/data-sources/remote_networks.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vmanilo) [reply docs/data-sources/remote_networks.md]: updated

File: docs/data-sources/remote_networks.md (hunk 1 above)
Comment (vmanilo) [reply docs/data-sources/remote_networks.md]: updated
